src/Build/Evaluation/IntrinsicFunctions.cs

```diff
@@ -350,6 +351,26 @@ internal static string ValueOrDefault(string conditionValue, string defaultValue
             }
         }
 
+        /// <summary>
+        /// Returns the string after converting all bytes to base 64 (alphanumeric characters plus '+' and '/'), ending in one or two '='.
+        /// </summary>
+        /// <param name="toEncode">String to encode in base 64.</param>
+        /// <returns></returns>
+        internal static string ConvertToBase64(string toEncode)
+        {
+            return Convert.ToBase64String(Encoding.Default.GetBytes(toEncode));
```

Comment:
How do you know the right encoding to use?

We talked about this offline: should probably be UTF-8.